.vscode/settings.json

```diff
@@ -17,7 +17,7 @@
     "**/.DS_Store": true,
     "**/.git": true,
     "**/node_modules": true,
-    "{build,dist-modern,dist}/": true
```

Comment:
Now that we consider the dist folder source code rather than "generated content, do not touch" it is useful to be able to search within it.